tests/test_export.py

```diff
@@ -0,0 +1,95 @@
+def test_export(pipeline):
+    """Test export to BossDB.
+
+    Please note that uploading data to BossDB via this pipeline requires an API
+    token which can be obtained by creating an account at
+    https://api.bossdb.io. You will also need resource manager permissions from the team at https://bossdb.org.
+    """
+    import datetime
+
+    subject = pipeline["subject"]
+    session = pipeline["session"]
+    scan = pipeline["scan"]
+    volume = pipeline["volume"]
+    bossdb = pipeline["bossdb"]
+
+    subject.Subject.insert1(
+        dict(
+            subject="subject1",
+            sex="M",
+            subject_birth_date="2023-01-01",
+            subject_description="Cellpose segmentation of volumetric data."),
+        skip_duplicates=True,
+    )
+
+    session_key = dict(
+        subject="subject1",
+        session_id=0,
+    )
+    session.Session.insert1(
+        dict(
+            session_key,
+            session_datetime=datetime.datetime.now(),
+        ),
+        skip_duplicates=True,
+    )
+
+    session.SessionDirectory.insert1(
+        dict(session_key, session_dir="subject1/session1"),
+        skip_duplicates=True,
+    )
+    scan.Scan.insert1(
+        dict(
+            session_key,
+            scan_id=0,
+            acq_software="ScanImage",
+        ),
+        skip_duplicates=True,
+    )
+    volume.Volume.populate()
+    key = (volume.Volume & "subject='subject1'").fetch1("KEY")
+    volume.SegmentationParamSet.insert_new_params(
+        segmentation_method="cellpose",
+        paramset_idx=2,
+        params=dict(
+            diameter=None,
```

Comment:
Should it be 8 or None?

```suggestion
            diameter=None,
```